lib/flappi/response_builder.rb

```diff
@@ -266,7 +266,7 @@ def link(*link_params)
                    link_params.first
                  else
                    keys = %i[key path]
-                   Hash[link_params.each_with_index.map { |p, idx| [keys[idx], p] }]
+                   link_params.each_with_index.map { |p, idx| [keys[idx], p] }.to_h
```

Comment:
Change makes sense, but was there a new cop/rule that complained about this? (and the other changes in this PR)

Reply:
Yep, due to [Style/HashConversion](https://docs.rubocop.org/rubocop/cops_style.html#stylehashconversion).

I have now marked all violated rubocop rules in this PR.